docs/OV_Runtime_UG/auto_device_selection.md

```diff
@@ -269,6 +269,26 @@ The `ov::hint::model_priority` property enables you to control the priorities of
 
 @endsphinxdirective
 
+### Execution Device Priority
```

Comment:
What's the differences between `ov::execution_devices` and `ov::runtime::Core::get_available_devices()`? This section seems not talking about priorities, but only "query runtime target devices". Should it be put under `#### Checking Available Devices` above?